openstudiocore/src/utilities/bcl/templates/EnergyPlusMeasure/tests/energyplus_measure_test.rb

```diff
@@ -4,7 +4,7 @@
 require_relative '../measure.rb'
 require 'fileutils'
 
-class EnergyPlusMeasureName_Test < Minitest::Test
```

Comment:
@nllong you should be aware of this code in OpenStudio::BCLMeasure that replaces text in the template when you make a new measure

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/bcl/BCLMeasure.cpp#L106